### PR TITLE
Send meta to mss [MAILPOET-2333]

### PR DESCRIPTION
**Pull request by amine-mp**
[MAILPOET-2333]

I added the following email types that were not mentioned in the specs:

**The sending method test email** 
```
email_type = 'sending_test'
subscriber_status = 'unknown'
subscriber_source = 'administrator'
```

**Preview email**
```
email_type = 'preview'
subscriber_status = 'unknown'
subscriber_source = 'administrator'
```

I have also used `administrator` as `subscriber_source` for stats notifications, new subscriber notifications, and WordPress transactional emails.

[MAILPOET-2333]: https://mailpoet.atlassian.net/browse/MAILPOET-2333

### CODE REVIEW

File: lib/API/JSON/v1/Mailer.php [hunk 1]
```diff
@@ -40,7 +45,10 @@ function send($data = []) {
         (isset($data['sender'])) ? $data['sender'] : false,
         (isset($data['reply_to'])) ? $data['reply_to'] : false
       );
-      $result = $mailer->send($data['newsletter'], $data['subscriber']);
+      $extra_params = [
+        'meta' => $this->mailerMetaInfo->getSendingTestMetaInfo(),
```
Comment (JanJakes) [inline lib/API/JSON/v1/Mailer.php]: This looks like a generic sending API class. I don't think we can hardcode here "sending test" meta info. Can we?

File: lib/Cron/Workers/SendingQueue/SendingQueue.php [hunk 2]
```diff
@@ -67,6 +72,11 @@ function process() {
       }
       // clone the original object to be used for processing
       $_newsletter = (object)$newsletter->asArray();
+      $options = $newsletter->options()->findMany();
+      if (!empty($options)) {
+        $options = array_column($options, 'value', 'name');
```
Comment (JanJakes) [inline lib/Cron/Workers/SendingQueue/SendingQueue.php]: Please use https://github.com/mailpoet/mailpoet/blob/master/lib/Util/ArrayColumn.php.

File: lib/API/JSON/v1/Mailer.php (hunk 1 above)
Comment (amine-mp) [reply lib/API/JSON/v1/Mailer.php]: I checked the code and found that we only call it to send the test email on from the settings page. I know that this may change in the future, but then what do you suggest as metadata for this email?

Comment (JanJakes) [reply lib/API/JSON/v1/Mailer.php]: @amine-mp Yeah, it is kind of dangerous to suppose this is called for tests only. Maybe we should extend the API with some parameters? Or maybe we can ask what others think on Slack?

File: lib/Subscribers/LinkTokens.php [hunk 3]
```diff
@@ -0,0 +1,39 @@
+<?php
```
Comment (JanJakes) [inline lib/Subscribers/LinkTokens.php]: Why is this file here and how it is related to this commit?
